sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_options.hpp

```diff
@@ -29,7 +29,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      * @brief Service Version used.
      *
      */
-    std::string ApiVersion{"7.5"};
+    std::string ApiVersion{"7.6-preview.2"};
```

Comment:
Is the expectation that we won't ship another Stable release before this goes stable?

Reply:
yeah, everyone is doing betas till then